app/src/test/java/edu/uc/kovaciad/slicetracker/DataUnitTest.kt

```diff
@@ -26,7 +26,7 @@ class DataUnitTest {
 //    }
 
     @Test
-    fun twoplustwo() {
+    fun twoPlusTwo() {
```

Comment:
This one is very minor, but I renamed the function to comply with Kotlin naming standards: https://kotlinlang.org/docs/coding-conventions.html#function-names